### DISCUSSION TITLE
Feature Request : dynamically display  the time interval in the tooltip while doign selection on a graph

### DISCUSSION THREAD

**Discussion by ouinouin** [grafana/grafana#38209] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
while doing a selection on a graph, it would be nice  to be able to see how much time represents the interval selected instead of just seing the date/time actually under the crosshair. thsi would allow to select like a precise interval of two hours starting at a particular time more easily.

What do you think about that.

**Comment by ivanahuckova** [comment]
Hello @ouinouin! Could you please describe in more detail what would be a good solution for you. Because currently, when you are in graph panel and you are zooming in, you can see the starting time in the tooltip. And when you start to drag it, you see also ending interval. 
![image](https://user-images.githubusercontent.com/30407135/105823507-86cd6f00-5fbd-11eb-8a31-b54dcc51d9a5.png)

**Comment by ouinouin** [comment]
Hi, @ivanahuckova 
im thinking about displaying the informations like this on the tooltip while dragging for selection, very often you need to select a precise interval of time, like this you can see the end date dynamically  and also dont have to to the maths in your head to have the correct interval of time. I think in ergonomy point of view its a good thing to have those 3 values (start date end date duration) under the eye while dragging.

![Capture d’écran du 2021-01-26 14-25-52](https://user-images.githubusercontent.com/6713759/105851300-2ea96380-5fe3-11eb-9448-89010cf46850.png)


edit : while looking on the timerange at the top of the page, i notice the tooltip at the top could also give the informations about the duration of the period 
![Capture d’écran du 2021-01-26 14-54-03](https://user-images.githubusercontent.com/6713759/105853894-7b426e00-5fe6-11eb-92eb-f3cc20d936f0.png)

**Comment by ivanahuckova** [comment]
Thank you for providing more information and context. Highly appreciated! :)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
